corehq/apps/users/views/__init__.py

```diff
@@ -458,6 +464,16 @@ def invitations(self):
 
     @property
     def page_context(self):
+        if (not self.can_restrict_access_by_location
+            and any(not role.permissions.access_all_locations
+                    for role in self.user_roles)
+        ):
+            messages.warning(self.request, _(
+                "This project has user roles that restrict data access by "
+                "organization, but the software plan no longer supports that. "
+                "Any users assigned to roles that are restricted in data access "
+                "by organization can no longer access this project.  Please "
+                "update the existing roles."))
```

Comment:
@sravfeyn I just attached a picture of what it looks like to the PR note

@sheelio I'm conflicted.  I agree that that's true if you think about it as a feature that you enable, but if you think of it as a permission, then it's weird for it to be negative.  I'd actually initially written it as a negative, but it felt weird to have a list of checkboxes enabling permissions, with one checkbox that removes permissions.  I'm happy to reconsider though, how would you want to phrase it?